project_testing/housing_characteristics/HVAC Has Shared System.tsv

```diff
@@ -1,102 +1,3 @@
-Dependency=Geometry Building Type RECS	Dependency=HVAC Cooling Type	Dependency=HVAC Heating Type	Option=Cooling Only	Option=Heating and Cooling	Option=Heating Only	Option=None
```

Comment:
I don't know if these dependencies are needed in project_testing. May need to test.

Reply:
Integrity checks pass without these dependencies.